utils/isValidGuess.ts

```diff
@@ -0,0 +1,5 @@
+function isValidGuess () {
+  return true
```

Comment:
This feature still needs implementing.
Need to:
* Get game mode from state, or pass as an arg
* Use that to grab the correct dictionary
* Check if the users word is in our dictionary